app/src/main/java/com/pd/snickers/MainActivity.java

```diff
@@ -81,6 +81,13 @@ public void onClick(View v) {
             }
         });
 
+        findViewById(R.id.button_love).setOnClickListener(new View.OnClickListener() {
```

Comment:
Could you also add a screenshot of this Snacker (our Chocobar) in the ReadMe

Reply:
> Could you also add a screenshot of this Snacker (our Chocobar) in the ReadMe

Yes, Added Done